server/docs/resources.yml

```diff
@@ -7,6 +7,14 @@
       - authCookie: []
       - sudoToken: []
     parameters:
+      - in: query
+        name: account
+        description: |
+          Filter resources by authenticated user status.
```

Comment:
❓ I don't think this _does_ filter by status. It filters by the source (hence my API suggestions above ☝️).